clang/test/CodeGenSYCL/inline_asm.cpp

```diff
@@ -0,0 +1,34 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -triple spir64-unknown-linux-sycldevice -S -emit-llvm -x c++ %s -o - | FileCheck %s
+
+inline namespace cl {
+  namespace sycl {
+    class kernel {};
+  }
+}
+
+using namespace cl::sycl;
+
+template <typename name, typename Func>
+__attribute__((sycl_kernel)) void kernel_single_task(Func kernelFunc) {
+  // CHECK: %[[ARRAY_A:[0-9a-z]+]] = alloca [100 x i32], align 4
+  // CHECK-NEXT: %0 = bitcast [100 x i32]* %[[ARRAY_A]] to i8*
```

Comment:
I don't think lifetime markers are worth testing.

Reply:
Also, these both use %0, which might change depending on naming conventions.  Those shouldn't be in the test.

Reply:
Thanks, fixed.